public_html/wp-content/themes/pattern-directory/css/components/_pattern-search.scss

```diff
@@ -1,20 +1,28 @@
+@mixin focus_styles {
```

Comment:
If we do want this to be a consistent focus style, maybe this mixin should be in `settings/`, so it can be used across the site? Or is this style meant just for the search?

Reply:
I've only seen it on search so I was hesitant for it to be shared. The styling seems to be designed specifically for display on a blue background but would probably also work everywhere. ❓

Reply:
I should have directed that question at @shaunandrews 🙂  Is this focus style just for the header search, or would it be repeated anywhere else on the site?